bin/node/runtime/src/lib.rs

```diff
@@ -2183,10 +2181,14 @@ impl_runtime_apis! {
 		}
 	}
 
-	impl pallet_staking_runtime_api::StakingApi<Block, Balance> for Runtime {
+	impl pallet_staking_runtime_api::StakingApi<Block, Balance, AccountId> for Runtime {
 		fn nominations_quota(balance: Balance) -> u32 {
 			Staking::api_nominations_quota(balance)
 		}
+
+		fn era_page_count(era: sp_staking::EraIndex, account: AccountId) -> sp_staking::PageIndex {
+			Staking::api_era_page_count(era, account)
+		}
```

Comment:
This API call is only for validator exposures so the naming could reflect this and be a bit more intuitive. `era_page_count` gives me the impression that this is returning the total pages for a whole era.

```suggestion
		fn validator_exposure_page_count(era: sp_staking::EraIndex, who: AccountId) -> sp_staking::PageIndex {
			Staking::api_era_page_count(era, who)
		}
```

Reply:
The storage item that page corresponds to is called `ErasStakersPaged` (which derives its name from the existing non-paged `ErasStakers`). What do you think about renaming it as `eras_stakers_page_count`?

Reply:
Renamed to eras_stakers_page_count to keep in line with how storage items are named.